### ISSUE TITLE
ansible deployment of detect_files and the encoding nodes

### ISSUE THREAD

**Issue by markvdb** [yoe/sreview#9]
Skeleton already added to https://github.com/FOSDEM/video/tree/master/software/ansible/playbooks/roles .

**Comment by yoe** [comment]
There are actually three things that need to be done in Ansible: 

1. The master server configuration (i.e., review.video.fosdem.org). This is necessary, but low priority for me atm; I intend to do this not before everything else is ready.
2. The streambackend servers need to be configured so they run the detect_files script. Otherwise, sreview will never see that need files have appeared. 
3. Transcode nodes need to be fully configured in Ansible so they install and start the necessary subset of sreview. This part needs to be finished before FOSDEM, so that adding or removing extra hosts to or from the setup can be done easily and flexibly as transcode requirements increase or decrease.

**Comment by yoe** [comment]
streambackend server config is now added to ansible, and that seems to work. Still to be done is the master server config and transcode nodes thing

**Comment by yoe** [comment]
Renaming this to the part of sreview that we need to have ansible-deployable before FOSDEM, so we can add it to the relevant milestone

**Comment by yoe** [comment]
This is now done in the FOSDEM/infrastructure repository. There are some manual bits to be done, mostly because review.video.fosdem.org is not ansible-managed and so I need to manually approve access to the encode* hosts for gridengine, but other than that it's all automated.
